package.json

```
@@ -4,10 +4,14 @@
   "description": "Official Top.gg Node SDK",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "npm ci && tsc",
+    "build": "tsc",
+    "build:ci": "npm i --also=dev && tsc",
     "docs": "npm run build && npm run md",
     "md": "jsdoc2md ./dist/structs/Api.js > ./docs/Api.md && jsdoc2md ./dist/structs/Webhook.js > ./docs/Webhook.md",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "lint": "eslint '*/**/*.ts'",
```

```suggestion
    "lint": "eslint 'src/*/**/*.ts'",
```
shouldn't this be done to make sure that you're not focusing lint on anything else like node_modules or dist?

(I'm also not sure if this is how you do it or if it's `src/**/*.ts`)

---

none of those folders should have typescript files in it but it might speed up search to scope it in src

---

declaration files?

---

well eslint by default won't check node_modules regardless but yeah maybe it might be a performance boost